src/main/java/Duke.java

```diff
@@ -1,10 +1,266 @@
+                    tasks.add(deadline);
+                } else if (input.startsWith("event ") && input.charAt(6) != 97) {
+                    String[] strings = input.split("/", 2);
+                    try {
+                        if (!input.contains("/") || strings[1].contains("/") || strings[1].length() < 1) {
+                            throw new InvalidInputException("You fool. You cannot create a task without a description, " +
+                                    "or you have formatted your description wrongly. Check your spaces and try again.");
+                        }
+                    } catch (InvalidInputException ex) {
+                        System.err.println(ex);
+                        continue;
+                    }
+                    String[] strings1 = strings[0].split(" ", 2);
+                    String[] strings2 = strings[1].split(" ", 2);
+                    Event event = new Event(strings1[1], strings2[1]);
+                    output = event.toString();
+                    tasks.add(event);
+                } else {
+                    // Check if there are any spaces first. if there are no spaces, means that it is either an invalid input or no such task
+
+                    // Level-5
+                    try {
+                        if ((input.startsWith("todo") || input.startsWith("deadline") || input.startsWith("event")) && !input.contains(" ")) {
+                            throw new InvalidInputException("You fool. You cannot create a task without a description, " +
+                                    "or you have formatted your description wrongly. Check your spaces and try again.");
+                        } else {
+                            try {
+                                throw new NoSuchTaskException("You have entered an invalid message. Please do not commit this idiocy again.");
+                            } catch (NoSuchTaskException ex) {
+                                System.err.println(ex);
+                                continue;
+                            }
+                        }
+                    } catch (InvalidInputException ex) {
+                        System.err.println(ex);
+                        continue;
+                    }
+
+                }
+                Duke.textBox("Fine. I've added this meaningless task to your list: ",
+                        " --> " + output,
+                        "Satisfied now? You have " + (tasks.size() + 1) + " items in your list. ");
+                //tasks.add(new Task(output));
+                //tasks.get(counter).status = tasks.get(counter).status.substring(4);
+            }
+        }
+
+    }
+
+    public static class Storage {
+
+        private String filePath;
+        private FileWriter fileWriter;
+        private JSONParser jsonParser;
+
+        public Storage(String filePath) {
+            this.filePath = filePath;
+        }
+
+        @SuppressWarnings("unchecked")
```

Comment:
Perhaps shift the @SupressWarnings to the exact line which you would like to suppress?